packages/langgraph-agent/src/scripts/loadVectorStore.ts

```diff
@@ -0,0 +1,31 @@
+import { loadPdfsFromDirectory } from "../azure/find_pdfs.js";
+import { updateEnv } from "../utils/update_env.js";
+import { getEmbeddingClient } from "../azure/embeddings.js";
+import { fileURLToPath } from "url";
+import path from "path";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const NORTHWIND_PDF_DIRECTORY = "../data";
+
+async function loadData(embeddings: any): Promise<void> {
+  let dataLoaded = process.env.NORTHWIND_PDF_LOADED;
+
+  if (!dataLoaded || dataLoaded === "true") {
+    const dirPath = path.join(__dirname, NORTHWIND_PDF_DIRECTORY);
+    await loadPdfsFromDirectory(embeddings, dirPath);
+    await updateEnv("NORTHWIND_PDF_LOADED", "true");
+  }
+}
+
+export async function main() {
+  const embeddings = getEmbeddingClient();
+  await loadData(embeddings);
+}
+
+main()
```

Comment:
nit: it's usually best to avoid mixing promises and async/await, this can be refactored to a try/catch with a top-level await